specs/refactoring_of_node_monitoring_flows_into_node_agent.adoc

```diff
@@ -0,0 +1,181 @@
+// vim: tw=79
+
+= Refactoring Of Node Monitoring Flows Into Node-Agent
+
+== Introduction
+
+This specification deals with detailing the work involved in merging the
+node-monitoring and node-agent services.
+
+Currently the responsibilities of node-monitoring are as follows:
+
+* It packages the collectd custom plugins
```

Comment:
yeah I think the only inhibitor to that is the our namespacing of the metrics which includes cluster-id. But yeah we can always get that as a configurable parameter and then facilitate its use...
But yes we need to finalize on where these plugins will go in other words how its packaged